stubs/flake8-2020/flake8_2020.pyi

```diff
@@ -0,0 +1,10 @@
+import ast
```

Comment:
Maybe we should add a comment to the beginning that mentions why it isn't possible to just put a `py.typed` file to the `flake8_2020` repository.